### ISSUE TITLE
wilder wilds compat (hollowed logs)

### ISSUE THREAD

**Issue by Thingydude32** [DawnTeamMC/Promenade#153]
### What is your idea about?

it would be definitely cool if this mod had a wilder wilds compat about the hollowed logs for the woods in promenade

**Comment by bangetto** [comment]
yeah, and also palm trees are in both of them so a compatibility would e great!
